src/azure-cli/setup.py

```diff
@@ -169,6 +170,9 @@
     ],
     packages=find_packages(exclude=["*.tests", "*.tests.*", "tests.*", "tests"]),
     install_requires=DEPENDENCIES,
+    extras_require={
+        ":python_version<'3.0'": ['antlr4-python2-runtime']
```

Comment:
We have dropped Python 2. We can safely remove this dependency. https://github.com/Azure/azure-cli/pull/11363